lib/iris/cube.py

```diff
@@ -1627,9 +1627,17 @@ def dtype(self):
     @property
     def fill_value(self):
         """
-        The fill value for the masked data array of this
-        :class:`~iris.cube.Cube`. If ``None``, the default NumPy fill value for
-        the dtype of the masked data array will be used.
+        A value suitable for filling in missing (i.e. masked) datapoints of
+        this :class:`~iris.cube.Cube` s data.
+
+        This can also be ``None``, meaning that no suitable fill value is
```

Comment:
Let's see if we can get to the situation of having a one-line description of this property and then also have most of the contents of these two paragraphs as one paragraph that follows it.

For example:
```
"""
A fill value for the data of the :class:cube.

This is a value suitable for filling masked points in this cube's data.
This may be None, meaning no suitable fill value has been defined...
"""
```